yass-components-react/src/components/Inline/stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import * as classNames from "../../../types/yass.json";
+
+import Box from "../Box";
+import Inline from "./index";
+import Stack from "../Stack";
+import Text from "../Text";
+
+export default {
+  title: "Inline",
+  component: Inline,
+  argTypes: {
+    children: {
+      control: "text",
+    },
+    gap: {
+      control: { type: "select" },
+      options: classNames["gap"],
+    },
+    justifyContent: {
+      control: { type: "select" },
+      options: classNames["justify-content"],
+    },
+    alignItems: {
+      control: { type: "select" },
+      options: classNames["align-items"],
+    },
+  },
+} as ComponentMeta<typeof Inline>;
+
+export const Gap: ComponentStory<typeof Inline> = (args) => (
```

Comment:
(zoomed out, obv. We already have too many scale tokens 😂 )
<img width="465" alt="image" src="https://user-images.githubusercontent.com/4335944/205900196-1e49029a-ea9e-41a1-9f06-a767b65b10fb.png">